templates/app/webpack-default/hooks/post-create.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const ejs = require('ejs');
+const fs = require('fs-extra');
+const path = require('path');
+const spawn = require('child_process').spawn; // eslint-disable-line security/detect-child-process
+
+exports.id = 'com.appcelerator.angular.post-create';
+
+exports.init = (logger, config, cli) => {
+	cli.on('create.post.app', async (creator, next) => {
+		const projectName = cli.argv.name;
+		const projectPath = path.join(cli.argv['workspace-dir'], projectName);
+		try {
+			await Promise.all([
+				copyGitIgnore(projectPath),
+				updatePackageJson(projectPath, { name: projectName }),
+				renderReadme(projectPath, { name: projectName }),
+				installDependencies(projectPath, logger)
+			])
+			next();
```

Comment:
This pattern comes up a lot in our unit tests, and after some weirdness I have moved to the pattern like so:
``` js
try {
  // do something
} catch (err) {
  return next(err);
}
next();
```

Otherwise technically the call to `next()` in the try block could throw an `Error` and trigger the `catch` block calling `next(err);` again.